doc/dnshealthchecks.md

```diff
@@ -1,26 +1,20 @@
 # DNS Health Checks
-DNS Health Checks are a crucial tool for ensuring the availability and reliability of your multi-cluster applications. Kuadrant offers a powerful feature known as DNSPolicy, which allows you to configure and verify health checks for DNS endpoints. This guide provides a comprehensive overview of how to set up, utilize, and understand DNS health checks.
+DNS Health Checks are a tool provided by some DNS Providers for ensuring the availability and reliability of your DNS Records and only publishing DNS Records that resolve to healthy workloads. Kuadrant offers a powerful feature known as DNSPolicy, which allows you to configure these health checks for all the managed DNS endpoints created as a result of that policy. This guide provides a comprehensive overview of how to set up, utilize, and understand these DNS health checks.
 
-## What are DNS Health Checks?
-DNS Health Checks are a way to assess the availability and health of DNS endpoints associated with your applications. These checks involve sending periodic requests to the specified endpoints to determine their responsiveness and health status. by configuring these checks via the [DNSPolicy](./dns.md), you can ensure that your applications are correctly registered, operational, and serving traffic as expected.
+## Supported Providers
+
+we currently only support AWS Route53 DNS Health checks.
```

Comment:
I think linking to these would be good so they can understand what will be created in their AWS account